### DISCUSSION TITLE
Attribute type Selects' values are not localisable

### DISCUSSION THREAD

**Discussion by genox** [swellstores/community#191] [Help] [answered]
Hi,

In options, tag items can be edited and translated after they appear in the horizontal list. Attributes on the other hand seem not to offer this option. The toggle switch for "Localised" is missing and tag list items can not be clicked in the same manner as options can.

<img width="930" alt="Screenshot 2023-03-31 at 14 01 11" src="https://user-images.githubusercontent.com/5611407/229114691-98be4bf8-5b32-4a5c-9ee1-6e6df7f9de8a.png">

This seems to be a bug or a missing feature. :)

**Comment by sags95** [comment, marked as answer]
Hey @genox, this does appear to be a gap. Let me check to see if this is already logged, otherwise I'll create a ticket for it so we can improve this.

**Reply by genox** [reply]
Cool, thank you
